lib/vyasa_web/live/display_manager/display_live.html.heex

```diff
@@ -25,6 +25,12 @@
             <.live_component module={@mode.control_panel_component} id="control_panel" mode={@mode} />
         </div>
 
+    <.live_component
```

Comment:
hoverune is now a live component which is the responsiblity of the DM directly. 

This allows emitting events from the live_component and handling it directly by the parent liveview (DM)